source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,331 @@
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindGtPOCO.cs
+      :language:  csharp
+
+   .. output::
+      :language: json
+      :visible:
+
+      { "_id" : 4, "make" : "Kiesel", "models" : ["Ares", "Vader", "Solo"], "establishedYear" : 2015, "rating" : null }
+
+The following example uses builders to create a query filter that matches all 
+documents where the ``establishedYear`` field is greater than ``1985``.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindGtBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Logical Operators
+-----------------
+
+Logical operators query data using logic applied to the results of two or more
+sets of search criteria. Common logical operators include:
+
+.. list-table::
+   :widths: 30 30 40
+   :header-rows: 1
+
+   * - Expression
+     - Builder
+     - Description
+
+   * - ``&&``
+     - ``And()``
+     - All sets of search criteria must be true.
+
+   * - ``||``
+     - ``Or()``
+     - At least one of set of search criteria must be true.
+
+For a full list of logical operators, see the :manual:`Logical
+Query Operators </reference/operator/query-logical/>` page. 
+
+The following example specifies a query filter as a parameter to the ``Find()`` 
+method, using the ``Guitar`` class. The query matches all documents where the
+``establishedYear`` field is greater than or equal to ``1985``, and the ``make``
+field is not equal to "Kiesel".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindAndPOCO.cs
+      :language:  csharp
+
+   .. output::
+      :language: json
+      :visible:
+
+      { "_id" : 3, "make" : "PRS", "models" : ["Silver Sky", "SE", "Custom"], "establishedYear" : 1985, "rating" : 9 }
+
+
+The following example uses builders to creates a query filter that matches all 
+documents where the ``establishedYear`` field is greater than or equal to ``1985``,
+and the ``make`` field is not equal to "Kiesel".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindAndBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 3, "make" : "PRS", "models" : ["Silver Sky", "SE", "Custom"], "establishedYear" : 1985, "rating" : 9 }
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Array Operators
+---------------
+
+Array operators query data based on the value or quantity of elements in an array
+field. Array operators include ``All()``, ``Any()``, and ``Size()``.
+
+.. note:: 
+
+   The {+driver-long+} translates the ``Any()`` operator to the MongoDB ``$elemMatch``
```

Comment:
Question/Issue: I believe the ``Any()`` specified here is a builder rather than an operator? Should this read 

```suggestion
   The ``Any()`` builder uses the ``$elemMatch`` query operator.
```